src/main/java/org/reactome/release/qa/common/QACheckerHelper.java

```diff
@@ -24,6 +24,8 @@
     
 	public static final String IS_NOT_NULL = "IS NOT NULL";
 	public static final String IS_NULL = "IS NULL";
+	private static List<String> skiplistDbIds = new ArrayList<>();
```

Comment:
I was wondering if the skip list db id list might be better as a member of the individual checks rather than in the QACheckerHelper?  Having it as a static member means it will be a single object shared among the checks.  This shouldn't be a problem if the checks are run serially and the skip lists are set at each check, but it might be problematic if the checks are run in parallel.  It might make more sense for each check to own its own skip list.  Maybe it could be a member in the common parent class of `AbstractQACheck`?

Reply:
That would make sense if different QA checks could have different skip lists.